webcomponents/tool/src/main/frontend/js/calendar/sakai-calendar.js

```diff
@@ -162,11 +177,9 @@ export class SakaiCalendar extends LionCalendar {
 
     return html`
 
+      <div>${this.i18n.days_message.replace("{0}", this.days)}</div>
```

Comment:
Shouldn't this kind of replacement be properly managed by the i18n system?

Reply:
If you use the tr function, yes, but I'm just happy with it this way. It's not too long and you know exactly what's going on.

Reply:
This would be the alternative, and it relies on the translations being loaded by a getproperties call:
${tr("calendar", "days_message", [this.days])}